services/README.md

```diff
@@ -0,0 +1,54 @@
+# A Quick Start Guide
+
+This guide provides instructions on how to start the SCX scheduler and check its logs.
+
+## Getting Started
+To start the SCX scheduler at boot, you need to run the systemd service as root. Here are the steps:
+
+- Enable the service:
+
+```
+systemctl enable scx_SCHEDNAME
+```
+
+- Start the service:
+
+```
+systemctl start scx_SCHEDNAME
+```
+
+Alternatively, you can use a shortened version of these commands:
+
+```
+systemctl enable --now scx_SCHEDNAME
+```
+
+- To check the status of the service, use the following command:
+
+```
+systemctl status scx_SCHEDNAME
+```
+
+## Checking Journald Logs
+
+The SCX schedulers do not log to the main journald. Instead, they save logs in a dedicated journald namespace.
```

Comment:
What do you think about:

The SCX schedulers do not log to the default journald namspace. Instead, they save logs in a dedicated `sched-ext` namespace.

Reply:
Fixed